frontend/README.md

```diff
@@ -6,18 +6,18 @@ The AML experiment client if the user facing part of the AML experiments. It mai
 -   Handles the experiment flow by showing views based on the data received from the AML experiment server
 -   Shows a user profile for the current experiment
 
-### Experiment views
+### Block views
 
--   To use a new experiment view, add it to the experiment rules in your AML experiment server.
+-   To use a new block view, add it to the experiment rules in your AML experiment server.
```

Comment:
Perhaps already adjust the README referring to the backend location as well. We're likely to forget it later.